pkg/controller/pagerdutyintegration/clusterdeployment_deleted.go

```diff
@@ -58,6 +58,9 @@ func (r *ReconcilePagerDutyIntegration) handleDelete(pdclient pd.Client, pdi *pa
 	}
 
 	ClusterID := cd.Spec.ClusterName
```

Comment:
```suggestion
	clusterID := cd.Spec.ClusterName
```
This variable shouldn't be exported.